README.md

```diff
@@ -1,8 +1,6 @@
-# Event::Source
+# EventSourcing
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/event/source`. To experiment with that code, run `bin/console` for an interactive prompt.
-
-TODO: Delete this and the text above, and describe your gem
+Simple PORO Event Sourcing framework, with Rails support. 
```

Comment:
I'm not sure that most people will know what "PORO" means. You could probably substitute "Ruby" everywhere you say "PORO".

This is the introduction section, you should probably also briefly describe what an "event sourcing framework" is and why you might need one. Just a paragraph or two.